app/views/modals/confirm.html

```diff
@@ -1,8 +1,8 @@
 <div class="modal-resource-action">
   <div class="modal-body">
     <h1>{{message}}</h1>
+    <p ng-if="details" ng-bind-html="details"></p>
```

Comment:
There is existing code that uses this dialog which might not be passing HTML sanitized values in details. This really need to be a separate property like `detailsMarkup` or something that makes it clear markup is expected.